index.bs

```diff
@@ -1182,9 +1182,9 @@ controls the maximum [=map/size=] of a [=trigger spec map=] for an
 controls how many [=attribution sources=] can be in the
 [=attribution source cache=] per [=attribution source/source origin=].
 
-<dfn>Randomized response epsilon</dfn> is a non-negative double that controls
-the randomized response probability of an [=attribution source=]. If [=automation local testing mode=] is true,
-this is `∞`.
+<dfn>Max settable event-level epsilon</dfn> is a non-negative double that
+controls the default and maximum values that a source registration can specify
+for the epsilon parameter used by [=compute the channel capacity of a source=].
```

Comment:
It is weird to highlight the channel capacity algorithm and not https://wicg.github.io/attribution-reporting-api/#obtain-a-randomized-source-response, which is the actual DP mechanism.